### PR TITLE
Revert yellow skull behavior to the old tibia style

### PR DESCRIPTION
**Pull request by Randomicoanom**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling].
- [x] I have read and understood the [contribution guidelines] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

**Protocol version**
8.60

### Changes Proposed

Change the default upstream TFS behavior (new tibia) to the old one 
https://github.com/otland/forgottenserver/issues/520
https://github.com/otland/forgottenserver/pull/562

**Issues addressed:** 

Player will receive a yellow skull and will be PZ Locked if fights back

### CODE REVIEW

File: src/player.cpp [hunk 1]
```diff
@@ -391,6 +391,11 @@ uint16_t Player::getClientIcons() const
 		}
 	}
 
+	if (pzLocked) {
+
```
Comment (nekiro) [inline src/player.cpp:395]: remove extra new line

**Review by nekiro — CHANGES_REQUESTED** [review]
allowFightBack isn't very descriptive though.
Better name would be, pzLockSkulledAttacker or something

File: src/configmanager.cpp [hunk 2]
```diff
@@ -234,6 +234,7 @@ bool ConfigManager::load()
 	boolean[REMOVE_WEAPON_AMMO] = getGlobalBoolean(L, "removeWeaponAmmunition", true);
 	boolean[REMOVE_WEAPON_CHARGES] = getGlobalBoolean(L, "removeWeaponCharges", true);
 	boolean[REMOVE_POTION_CHARGES] = getGlobalBoolean(L, "removeChargesFromPotions", true);
+	boolean[PZLOCK_SKULLED_ATTACKER] = getGlobalBoolean(L, "pzLockSkulledAttacker", true);
```
Comment (nekiro) [inline src/configmanager.cpp:237]: Sorry for being picky, but change name to pzLockSkullAttacker, skulled isn't exactly right, mb for suggesting it

**Review by nekiro — CHANGES_REQUESTED** [review]

File: src/luascript.cpp [hunk 3]
```diff
@@ -1895,6 +1895,7 @@ void LuaScriptInterface::registerFunctions()
 	registerEnumIn("configKeys", ConfigManager::REMOVE_WEAPON_AMMO)
 	registerEnumIn("configKeys", ConfigManager::REMOVE_WEAPON_CHARGES)
 	registerEnumIn("configKeys", ConfigManager::REMOVE_POTION_CHARGES)
+    registerEnumIn("configKeys", ConfigManager::PZLOCK_SKULL_ATTACKER)
```
Comment (EPuncker) [inline src/luascript.cpp:1898]:
```suggestion
	registerEnumIn("configKeys", ConfigManager::PZLOCK_SKULL_ATTACKER)
```

File: src/configmanager.cpp [hunk 4]
```diff
@@ -234,6 +234,7 @@ bool ConfigManager::load()
 	boolean[REMOVE_WEAPON_AMMO] = getGlobalBoolean(L, "removeWeaponAmmunition", true);
 	boolean[REMOVE_WEAPON_CHARGES] = getGlobalBoolean(L, "removeWeaponCharges", true);
 	boolean[REMOVE_POTION_CHARGES] = getGlobalBoolean(L, "removeChargesFromPotions", true);
+	boolean[PZLOCK_SKULL_ATTACKER] = getGlobalBoolean(L, "pzLockSkullAttacker", true);
```
Comment (EPuncker) [inline src/configmanager.cpp:237]:
```suggestion
	boolean[PZLOCK_SKULL_ATTACKER] = getGlobalBoolean(L, "pzLockSkullAttacker", false);
```
if default is false in config.lua we should have it false here as well

File: src/player.cpp [hunk 5]
```diff
@@ -391,6 +391,10 @@ uint16_t Player::getClientIcons() const
 		}
 	}
 
+	if (pzLocked) {
+		icons |= ICON_REDSWORDS;
```
Comment (EPuncker) [inline src/player.cpp:395]: I'm pretty sure redswords icon didn't existed in 8.6, but I might be wrong

File: src/luascript.cpp (hunk 3 above)
Comment (Randomicoanom) [reply src/luascript.cpp:1898]: that's weird i don't know why it end up that way

**Pull request merged**